test/framework/toy_build.py

```diff
@@ -2390,6 +2392,8 @@ def test_fix_shebang(self):
             "   'echo \"#!/usr/bin/perl -w\\n# test\" > %(installdir)s/bin/t4.pl',",
             # space after #! + 'env perl5'
             "   'echo \"#!/usr/bin/env perl5\\n# test\" > %(installdir)s/bin/t5.pl',",
+            # no shebang perl
+            "   'echo \"# test\" > %(installdir)s/bin/t6.pl',",
```

Comment:
Also add a test which has `#!/bin/bash` as shebang, to make sure you don't break scripts that do have a shebang?

Reply:
Done